src/cplscheme/tests/CompositionalCouplingSchemeTest.cpp

```diff
@@ -223,12 +223,10 @@ BOOST_AUTO_TEST_SUITE(DummySchemeCompositionTests)
 BOOST_AUTO_TEST_CASE(testDummySchemeCompositionExplicit2)
 {
   PRECICE_TEST(1_rank, Require::Events);
-  int               numberIterations = 1;
-  int               maxTimesteps     = 10;
-  PtrCouplingScheme scheme1(
-      new tests::DummyCouplingScheme(numberIterations, maxTimesteps));
-  PtrCouplingScheme scheme2(
-      new tests::DummyCouplingScheme(numberIterations, maxTimesteps));
+  int                         numberIterations = 1;
+  int                         maxTimeWindows   = 10;
```

Comment:
All tests are about the number of performed windows (in this case equals time steps, because no subcycling). But I would call them windows, because this is also the name I would use in the constructor of the scheme.

Reply:
yes